File: Java/Penayangan.java
public class Penayangan { 
    // Atribut-atribut untuk menyimpan informasi penayangan
    private String judulFilm;
    private String tanggal;
    private String jam;
    private String studio;
    private int hargaTiket;

    // Konstruktor default
    public Penayangan() {
        this.judulFilm = "";
        this.tanggal = "";
        this.jam = "";
        this.studio = "";
        this.hargaTiket = 0;
    }

    // Konstruktor dengan parameter
    public Penayangan(String judulFilm, String tanggal, String jam, String studio, int hargaTiket) {
        this.judulFilm = judulFilm;
        this.tanggal = tanggal;
        this.jam = jam;
        this.studio = studio;
        this.hargaTiket = hargaTiket;
    }

    // Getter & Setter
    public String getJudulFilm() {
        return judulFilm;
    }

    public void setJudulFilm(String judulFilm) {
        this.judulFilm = judulFilm;
    }

    public String getTanggal() {
        return tanggal;
    }

    public void setTanggal(String tanggal) {
        this.tanggal = tanggal;
    }

    public String getJam() {
        return jam;
    }

    public void setJam(String jam) {
        this.jam = jam;
    }

    public String getStudio() {
        return studio;
    }

    public void setStudio(String studio) {
        this.studio = studio;
    }

    public int getHargaTiket() {
        return hargaTiket;
    }

    public void setHargaTiket(int hargaTiket) {
        this.hargaTiket = hargaTiket;
    }

    public void tampilkan() {
        System.out.println("Judul Film            : " + judulFilm);
        System.out.println("Tanggal (YYYY-MM-DD)  : " + tanggal);
        System.out.println("Jam (HH:MM)           : " + jam);
        System.out.println("Studio                : " + studio);
        System.out.println("Harga Tiket           : " + hargaTiket);
    }
}

File: Cpp/Penayangan.cpp
#include <string>
#include <iostream>

using namespace std;

class Penayangan{
    private: // Atribut (data member) dari class Penayangan
        string judulFilm;
        string tanggal;
        string jam;
        string studio;
        int hargaTiket;

    public:
		// Konstruktor default dan konstruktor dengan parameter
        Penayangan(){
            judulFilm = "";
            tanggal = "";
            jam = "";
            studio = "";
            hargaTiket = 0;
        }

		// Konstruktor dengan parameter
        Penayangan(string judulFilmP, string tanggalP, string jamP, string studioP, int hargaTiketP){
            judulFilm = judulFilmP;
            tanggal = tanggalP;
            jam = jamP;
            studio = studioP;
            hargaTiket = hargaTiketP;
        }

		// Setter dan getter untuk setiap atribut
		// Setter untuk judulFilm
        void setJudulFilm(string judulFilmP){
            judulFilm = judulFilmP;
        }

		// Getter untuk judulFilm
        string getJudulFilm(){
            return judulFilm;
        }

		// Setter untuk tanggal
        void setTanggal(string tanggalP){
            tanggal = tanggalP;
        }

		// Getter untuk tanggal
        string getTanggal(){
            return tanggal;
        }

		// Setter untuk jam
        void setJam(string jamP){
            jam = jamP;
        }

		// Getter untuk jam
        string getJam(){
            return jam;
        }

		// Setter untuk studio
        void setStudio(string studioP){
            studio = studioP;
        }

		// Getter untuk studio
        string getStudio(){
            return studio;
        }

		// Setter untuk hargaTiket
        void setHargaTiket(int hargaTiketP){
            hargaTiket = hargaTiketP;
        }

		// Getter untuk hargaTiket
        int getHargaTiket(){
            return hargaTiket;
        }

		// Method untuk menampilkan informasi penayangan
		void tampilkan(){
			cout << "Judul Film  : " << judulFilm << endl;
			cout << "Tanggal     : " << tanggal << endl;
			cout << "Jam         : " << jam << endl;
			cout << "Studio      : " << studio << endl;
			cout << "Harga Tiket : " << hargaTiket << endl;
		}

		// Destructor
        ~Penayangan(){}
};